readthedocs/templates/projects/project_advanced.html

```diff
@@ -13,5 +13,10 @@
 {% block project_edit_content_header %}{% trans "Advanced Settings" %}{% endblock %}
 
 {% block project_edit_content %}
-  {% crispy form %}
+<form method="post" action=".">{% csrf_token %}
+  {{ form|crispy }}
+  <p>
+    <input style="display: inline;" type="submit" value="{% trans "Save" %}">
+  </p>
```

Comment:
Not a big issue but FYI: this element structure is unnecessary. `input` is a block element (`display: block`) by default, so changing the display type of `input` and wrapping in a `p` block element is the same thing as:

```suggestion
  <input type="submit" value="{% trans "Save" %}">
```

This avoids hardcoding styles outside stylesheets

Reply:
Wow! Thanks for the explanation. I still have no idea how HTML and CSS works 😂 

BTW, I didn't write this code by myself, I just copied from another template. I think it was from here: https://github.com/readthedocs/readthedocs.org/blob/d23bd1315a8e266ffcb662c89521c9f5b6671fc2/readthedocs/templates/projects/project_edit.html#L15-L27

Probably the `p` makes sense there because it has another `a` inside with to "Delete project".

Reply:
Hah yeah this is exactly what I assumed happened. It's extra typing unless you specifically need the wrapping paragraph tag (like having inline links).